Documentation/ceph-cluster-crd.md

```diff
@@ -116,7 +116,7 @@ Below are the settings available, both at the cluster and individual node level,
 - `directories`:  A list of directory paths that will be included in the storage cluster. Note that using two directories on the same physical device can cause a negative performance impact.
   - `path`: The path on disk of the directory (e.g., `/rook/storage-dir`).
   - `config`: Directory-specific config settings. See the [config settings](#osd-configuration-settings) below.
-- `location`: Location information about the cluster to help with data placement, such as region or data center.  This is directly fed into the underlying Ceph CRUSH map.  More information on CRUSH maps can be found in the [ceph docs](http://docs.ceph.com/docs/master/rados/operations/crush-map/).
+- `location`: Location information about the cluster to help with data placement, such as region or data center.  This is directly fed into the underlying Ceph CRUSH map. The format of this field is `string`. For example, to add datacenter location information, set this field to `rack=rack1`.  More information on CRUSH maps can be found in the [ceph docs](http://docs.ceph.com/docs/master/rados/operations/crush-map/).
```

Comment:
To note, I think it is enough that the "extra" description is added here as the Ceph block pool CRD and Ceph Filesystem CRD link to this page in the end.

Reply:
does it hurt though?

Reply:
Mhh no it wouldn't hurt.

Reply:
i also added a comment to cephfs crd